src/main/java/com/faforever/client/map/MapService.java

```diff
@@ -303,7 +306,12 @@ public MapBean readMap(Path mapFolder) throws MapLoadException {
   @Cacheable(value = CacheNames.MAP_PREVIEW, unless = "#result == null")
   public Image loadPreview(String mapName, PreviewSize previewSize) {
     if (mapGeneratorService.isGeneratedMap(mapName)) {
-      return mapGeneratorService.getGeneratedMapPreviewImage();
+      try {
+        BufferedImage image = ImageIO.read(forgedAlliancePreferences.getCustomMapsDirectory().resolve(mapName).resolve(mapName + "_preview.png").toFile());
```

Comment:
Why do we have an AWT image only to convert it to an java fx image later, makes little sense to me pls try do this the java fx way...

Reply:
yeah i was just lazy and didnt change the import settings but will do so